src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -205,7 +206,7 @@ public class RunResult
     {
         public bool Failed { get; set; }
 
-        public Exception Exception { get; set; }
+        public ExceptionDispatchInfo Exception { get; set; }
```

Comment:
I'm OK with leaving it is, just wanted to ask the question to see what people thought, which is why I didn't mark this as "Request Changes".